diff --git a/lib/audio/gbchannel.js b/lib/audio/gbchannel.js
index 2e327e9..4b97652 100644
--- a/lib/audio/gbchannel.js
+++ b/lib/audio/gbchannel.js
@@ -13 +13 @@ import toWav from 'audiobuffer-to-wav';
-const DEFAULT_AUDIO_LATENCY_IN_MILLI = 100;
+const DEFAULT_AUDIO_LATENCY_IN_MILLI = 35;
diff --git a/lib/wasmboy/worker/update.js b/lib/wasmboy/worker/update.js
index e7ac5bd..90e3746 100644
--- a/lib/wasmboy/worker/update.js
+++ b/lib/wasmboy/worker/update.js
@@ -27 +27 @@ import { transferGraphics } from './graphics/transfer';
-const MAX_AUDIO_LATENCY = 0.25;
+const MAX_AUDIO_LATENCY = 0.08;
@@ -30 +30 @@ const MAX_AUDIO_LATENCY = 0.25;
-const AUDIO_BUFFER_SIZE = 1024;
+const AUDIO_BUFFER_SIZE = 512;
